test/oc_tab_reporter.erl

```diff
@@ -17,15 +17,19 @@
 %%%-----------------------------------------------------------------------
 -module(oc_tab_reporter).
 
--behaviour(oc_reporter).
+-behaviour(gen_event).
 
 -export([init/1,
-         report/2]).
+         handle_call/2,
+         handle_event/2]).
 
 init(_) ->
-    application:get_env(opencensus, tab_reporter, #{}).
+    {ok, application:get_env(opencensus, tab_reporter, #{})}.
```

Comment:
Table ID should be passed as a configuration value to the `oc_tab_reporter` instead of using application env.